config/environment.rb

```diff
@@ -11,6 +11,8 @@
 require 'logger'
 
 require 'sinatra'
+require 'rack-flash'
```

Comment:
Don't you have to require rack-flash3` What does the gem documentation say?

Reply:
Here in the https://github.com/vast/sinatra-redirect-with-flash , in the ##Example section, it puts -require 'rack-flash'- without 3. It's a bit confusing because in the ##Installation section you have to put -gem 'rack-flash3'- as a dependency, but that's why I left it as rack-flash only in config/environment.rb

Reply:
Okay